plugwise/helper.py

```
@@ -655,14 +637,19 @@ def _get_plugwise_notifications(self) -> None:
                 )
 
     def _get_actuator_functionalities(
-        self, xml: etree, device: DeviceData, data: DeviceData
+        self, xml: etree, device: DeviceZoneData, data: DeviceZoneData
     ) -> None:
         """Helper-function for _get_measurement_data()."""
+        if appliance.find("type").text not in ACTUATOR_CLASSES:
```

Ruff

Undefined name `appliance`

(F821)